app/models/facility.rb

```diff
@@ -101,8 +102,20 @@ def to_s
     "#{name} (#{abbreviation})"
   end
 
+  def problem_order_details
```

Comment:
I think this naming scheme is kind of confusing. I would expect `problem_order_details` to be all order details, not to exclude anything. I was actually fine with using `reservations` and `non_reservations`.